meteor/server/api/showStyles.ts

```diff
@@ -75,50 +75,94 @@ export async function insertShowStyleBaseInner(organizationId: OrganizationId |
 		_rundownVersionHash: '',
 	}
 	ShowStyleBases.insert(showStyleBase)
-	await insertShowStyleVariantInner(showStyleBase, 'Default')
+
+	const showStyleVariant: ShowStyleVariant = {
```

Comment:
This could be DRY:ed by instead calling `createDefaultShowStyleVariant(context, showStyleBase._id, 'Default')`.

Also, the default rank is set to 500 here, but is 0 in `createDefaultShowStyleVariant`.